.github/workflows/build-service.yml

```diff
@@ -18,8 +18,25 @@ on:
 permissions:
   packages: write
 
+env:
+  STACK_TAG_LIST: '["latest"]'
+
 jobs:
+  matrix-gen:
+    # This job exists solely because fromJSON() cannot directly accept
+    # env.STACK_TAG_LIST for some reason.
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - id: set-matrix
+        run: echo "matrix=${STACK_TAG_LIST}" >> $GITHUB_OUTPUT
+
   test-service:
+    needs: matrix-gen
+    strategy:
+      matrix:
+        stackTag: ${{ fromJSON(needs.matrix-gen.outputs.matrix) }}
```

Comment:
Yes, I expect STACK_TAG_LIST to eventually have more than one element: "latest" as well as certain stack versions that we want to build against (e.g. because they avoid recent bugs or incompatibilities).  That's why the matrix; the `matrix-gen` job is so that the list can be set once instead of repeated for the test and build jobs.

Reply:
But `STACK_TAG_LIST` will have a fixed value in the short term? I'm having trouble seeing what the use case is.